### ISSUE TITLE
\ before global namespace functions causes an escaping function error.

### ISSUE THREAD

**Issue by GaryJones** [WordPress-Coding-Standards/WordPress-Coding-Standards#933]
Related to #764.

This method is in a namespaced class:

```php
public function do_footer_nav() {
	echo \wp_kses_post(
		\genesis_get_nav_menu(
			[
				'menu_class'     => 'menu genesis-nav-menu menu-footer',
				'theme_location' => 'footer',
			]
		)
	);
}
```

The presence of the `\` before the function calls from the global namespace, means that an error of `Expected next thing to be an escaping function, ... not "\"` appears.

I've not looked into the change needed, but if the sniff could allow an optional `\` before those function checks, that would be great :-)

Using fully-qualified function calls is faster. Details and benchmarks are [here](https://github.com/Roave/FunctionFQNReplacer). This is from an inspection from the PHP Inspections extension for PhpStorm.
